R/theme-elements.r

```diff
@@ -16,7 +16,7 @@
 #' @param colour,color Line/border colour. Color is an alias for colour.
 #' @param size Line/border size in mm; text size in pts.
 #' @param inherit.blank Should this element inherit the existence of an
-#'   `element_blank` among its parents? If `TRUE` the existence of
+#'   `element_blank()` among its parents? If `TRUE` the existence of
```

Comment:
Not sure about this change and a few like it. `element_blank()` is a function that creates an object of type `element_blank`. Are we referring about the function or the object here?

Reply:
Thanks, good catch. Removed here and in one other instance.